Formula/r/rhai.rb

```diff
@@ -0,0 +1,48 @@
+class Rhai < Formula
+  desc "Embedded scripting language for Rust"
+  homepage "https://rhai.rs/"
+  url "https://github.com/rhaiscript/rhai/archive/refs/tags/v1.21.0.tar.gz"
+  sha256 "11b1303830b95efe153c8fcab25628b8304b31f4c2b8ecd0791979800db19e49"
+  license any_of: ["Apache-2.0", "MIT"]
```

Comment:
can you also add it to the autobump?

```suggestion
  license any_of: ["Apache-2.0", "MIT"]
  head "https://github.com/rhaiscript/rhai.git", branch: "main"
```